HarryPotter/Screens/House/ViewModel/HouseViewModel.swift

```diff
@@ -16,8 +16,25 @@ struct Character {
     let alive: Bool
 }
 
-protocol HouseViewModelDelegate {
-    func refreshView()
+enum HouseOfHogwart: Int {
+    case gryffindor = 0
+    case slytherin = 1
+    case ravenclaw = 2
+    case hufflepuff = 3
+    
+    var apiName: String {
```

Comment:
You already have a `House` enum created. What you can do here in HouseViewModel, is... (instead of creating a second similar object HouseOfHogward)

```
extension House {
   var apiName: String { ... }
}